bananalyzer/data/fetch_schemas.py

```diff
@@ -1,31 +1,31 @@
 import json
 from typing import Dict, Any
 
-from bananalyzer.data.schemas import FetchId
+from bananalyzer.data.schemas import SchemaName
 from bananalyzer.data.examples import get_examples_path
 
 """
-This file contains mapping of fetch_id to fetch schema to avoid duplicate schemas in examples.json
+This file maps a schema_name to a JSON schema or goal string.
 """
 
 
-def get_fetch_schema(fetch_id: FetchId) -> Dict[str, Any]:
+def get_fetch_schema(schema_name: str) -> Dict[str, Any]:
```

Comment:
Is there a way we could maintain typechecking the schema string as `SchemaName` but still handle the default case cleanly (and in `Example.set_schema_and_goal`)?

Reply:
not really, we'd probably need to codegen based on the keys of that json but its not really worth it

Reply:
ok cool then